src/graphics.rs

```diff
@@ -95,18 +95,16 @@ where
     }
 
     fn clear(&mut self, color: Self::Color) -> Result<(), Self::Error>
-    where
-        Self: Sized,
```

Comment:
I don't remember making this change. I don't see any good reason for the including these lines, so I left it in.

Reply:
I think Self: Sized is implied either way?

Reply:
Yes, I think it is.